cmd/config.go

```diff
@@ -102,6 +106,30 @@ func runValidateConfig(commandContext *cmdctx.CmdContext) error {
 	return errors.New("App configuration is not valid")
 }
 
+func runEnvConfig(ctx *cmdctx.CmdContext) error {
+	secrets, err := ctx.Client.API().GetAppSecrets(ctx.AppName)
+	if err != nil {
+		return err
+	}
+
+	cfg, err := ctx.Client.API().GetConfig(ctx.AppName)
+	if err != nil {
+		return err
+	}
+
+	vars := cfg.Definition["env"].(map[string]interface{})
```

Comment:
You should make sure `cfg.Definition` is not nil, then make sure it has an `env` key, then make sure the env key is a `map[string]string` to prevent a panic on bad data.